src/control/server/ctl_storage_rpc_test.go

```diff
@@ -299,10 +274,10 @@ func TestServer_CtlSvc_StorageScan_PreEngineStart(t *testing.T) {
 			log, buf := logging.NewTestLogger(t.Name())
 			defer common.ShowBufferOnFailure(t, buf)
 
-			tCfg := storage.NewTierConfig().WithBdevClass(storage.ClassNvme.String())
-			if !tc.noBdevsInCfg {
```

Comment:
Should remove this from the tc struct as well

Reply:
done